trulens_eval/examples/expositional/models/azure_openai.ipynb

```diff
@@ -29,7 +29,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.            system_prompt = system_prompt.replace(
</pre><p>Where is the actual LLM call happeningn here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/847/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Added `generate_score_and_reasons` method for COT methods and refactored to use it here.